packages/protoplugin/package.json

```diff
@@ -37,7 +37,7 @@
   "dependencies": {
     "@bufbuild/protobuf": "0.1.1",
     "@typescript/vfs": "^1.4.0",
-    "typescript": "4.5.2"
+    "typescript": "4.8.4"
```

Comment:
We should keep this pinned at the older version. We deliberately don't want to keep this up to date, because updating the TypeScript compiler can change to transpiled code, and we want the generated code to be stable. 

https://github.com/bufbuild/protobuf-es/blob/5ae3246385fec75f80a9cb058f9c8abc48912611/packages/protoplugin/src/ecmascript/transpile.ts#L56-L58